festflow/views.py

```diff
@@ -109,3 +112,30 @@ def complete_profile(request):
     context['backend'] = backend
     context['profile_form'] = profile_form
     return render(request, 'festflow/complete_profile.html', context)
+
+
+def subscribe(request):
+    context = {}
+
+    if request.method == 'POST':
+        form = SubscriptionForm(request.POST)
+        to_addr = request.POST.get('contact_email', '')
+        if Subscription.objects.filter(contact_email=to_addr).count():
```

Comment:
But that thread suggested something as old as 2009, hasn't Django paced up to provide a much better way? With exists()?